Tests/FoundationTests/NSAttributedStringExtensionsTests.swift

```diff
@@ -30,7 +30,7 @@ final class NSAttributedStringExtensionsTests: XCTestCase {
 	func testUnderlined() {
 		let string = NSAttributedString(string: "Underlined")
 		let out = string.underlined
-		let attributes = out.attributes(at: 0, effectiveRange: nil)
```

Comment:
@nathanbacon Mind if we change these back? I don't want the unit tests of 1 extension to depend on another extension. All SwifterSwift extensions should be self sufficient

Reply:
@SD10 I see, taken care of.